core/lib/spree/core.rb

```diff
@@ -81,6 +81,7 @@ class DestroyWithOrdersError < StandardError; end
 require 'spree/service_module'
 
 require 'spree/core/importer'
+require 'spree/core/filters'
```

Comment:
@bbonislawski `Filters` are kind of ambiguous - maybe sth like `CollectionFilters` ?

Reply:
Changed